src/box/iproto.cc

```diff
@@ -63,6 +62,8 @@
 #include "execute.h"
 #include "errinj.h"
 #include "tt_static.h"
+#include "assoc.h"
```

Comment:
For what purpose you add this header? I don't see anything that uses it.

Reply:
fixed